src/buildifier/buildifier.ts

```diff
@@ -164,6 +164,12 @@ export function getDefaultBuildifierExecutablePath(): string {
   // just use "buildifier" as the default and get it from the system PATH.
   const bazelConfig = vscode.workspace.getConfiguration("bazel");
   const buildifierExecutable = bazelConfig.get<string>("buildifierExecutable");
+  if (vscode.workspace.rootPath !== undefined) {
+    buildifierExecutable.replace(
+      "${workspaceFolder}",
+      vscode.workspace.rootPath
```

Comment:
Is the behavior of this consistent for a trailing (or lack of trailing) `/`'?

i.e. - should folks use `${workspaceFolder}/something` or `${workspaceFolder}something`?

Maybe tweak https://github.com/bazelbuild/vscode-bazel/blob/master/package.json#L99 to document this support?

Reply:
Folks should use `${workspaceFolder}/something` - and this is consistent with other usage of `${workspaceFolder}` in bazel